### PR TITLE
@ table cell can render UiComponent

### PR DESCRIPTION
**Pull request by dominikvoda**

### CODE REVIEW

File: src-php/Table/Ui/Table.php [hunk 1]
```diff
@@ -152,29 +152,29 @@ private function getCellRenderFunction(string $key): callable
...
+        return static function (
             CellData $cellData,
             RowData $rowData,
             ColumnDefinition $columnDefinition,
             TableIterator $iterator
         ): Cell {
             $value = $cellData->getValue();
             if (is_int($value)) {
-                $value = (string)$value;
+                return new Cell((string)$value, $columnDefinition);
```
Comment (michalbrauner) [inline src-php/Table/Ui/Table.php:163]: what about `is_float` too?

File: src-php/Table/Ui/__snapshots__/table.html [hunk 2]
```diff
@@ -22,6 +22,13 @@
                 <span class="Link__Text">Delete</span>
             </a></td>
         </tr>
+        <tr>
+            <td>Prokop</td>
+            <td><div class="be-icon-agent"></div></td>
+            <td style="width: 50%; max-width: 50%; text-align: right;"><a class="Link__Link Link__Blue" href="http://yolo?id=3">
+                <span class="Link__Text">Delete</span>
```
Comment (michalbrauner) [inline src-php/Table/Ui/__snapshots__/table.html:29]: where is rendered age and why is there button Delete? You added age

Comment (dominikvoda) [reply src-php/Table/Ui/__snapshots__/table.html:29]: there is rendering callback defined in test, it is just extended test to demosntrate rendering of UiComponent in form of Icon

File: src-php/Table/Ui/Table.php (hunk 1 above)
Comment (dominikvoda) [reply src-php/Table/Ui/Table.php:163]: I don't think you want to render float as is...you want to control decimal numbers etc.

**Pull request merged**